### PR TITLE
Adding tests for transaction trace

### PR DESCRIPTION
**Pull request by vickumar1981**
- Adds serialize/deserialize methods for transaction trace
- Adds tests for transaction trace
- Adds serialize/deserialize methods for `transaction_trace_msg`

### CODE REVIEW

**Review by heifner — APPROVED** [review]
Are you going to add transaction_trace_msg in this PR a diff one?

**Review by heifner — APPROVED** [review]

**Review by heifner — APPROVED** [review]

**Comment by vickumar1981** [comment]
@wynvandevb1

**Review by opi-smccoole — APPROVED** [review]
These changes look fine to me.  Probably want to add @GonnaGitYou

File: src/main/java/one/block/eosiojavaabieosserializationprovider/AbiEosSerializationProviderImpl.java [hunk 1]
```diff
@@ -207,6 +207,50 @@ public String serializeTransaction(String json) throws SerializeTransactionError
...
+     * @return - Serialized hex string representing the transaction trace JSON.
+     * @throws SerializeTransactionError - A serialization error is thrown if there are any exceptions during the
+     *      * conversion process.
+     */
+    @NotNull
+    public String serializeTransactionTrace(String json) throws SerializeTransactionError {
+        try {
+            String abi = getAbiJsonString("ship.abi.json");
+            AbiEosSerializationObject serializationObject = new AbiEosSerializationObject(null,
+                    "", "transaction_trace", abi);
```
Comment (wynvandevb1) [inline src/main/java/one/block/eosiojavaabieosserializationprovider/AbiEosSerializationProviderImpl.java:223]: perhaps pull those 2 above (abi file and transation_trace) values there and below into static vars

**Pull request merged**